docs/en/administration/query_profile_details.md

```diff
@@ -0,0 +1,1366 @@
+
+##### BytesSent
+
+Description: Size of sent data.
+
+##### BytesUnsent
+
+Description: Size of unsent data. This metric is non-zero when there is a short-circuit logic; otherwise, it is zero.
+
+##### BytesPassThrough
+
+Description: Size of data not transmitted over the network when the destination node is the current node, i.e., the size of PassThrough data. Enabled by `enable_exchange_pass_through`.
+
+##### PassThroughBufferPeakMemoryUsage
+
+Description: Peak memory usage of the PassThrough Buffer.
+
+##### CompressTime
+
+Description: Compression time.
+
+##### CompressedBytes
+
+Description: Size of compressed data.
+
+##### OverallThroughput
+
+Description: Throughput rate.
+
+##### NetworkTime
+
+Description: Time taken for data packet transmission (excluding post-reception processing time).
+
+##### NetworkBandwidth
+
+Description: Estimated network bandwidth.
+
+##### WaitTime
+
+Description: Waiting time due to a full sender queue.
+
+##### OverallTime
+
+Description: Total time for the entire transmission process, i.e., from sending the first data packet to confirming the correct reception of the last data packet.
+
+##### RpcAvgTime
+
+Description: Average time for RPC.
+
+##### RpcCount
+
+Description: Total number of RPCs.
+
+#### Exchange Source Operator
+
+##### RequestReceived
+
+Description: Size of received data packets.
+
+##### BytesReceived
+
+Description: Size of received data.
+
+##### DecompressChunkTime
+
+Description: Time taken to decompress chunks.
+
+##### DeserializeChunkTime
+
+Description: Time taken to deserialize chunks.
+
+##### ClosureBlockCount
+
+Description: Number of blocked Closures.
+
+##### ClosureBlockTime
+
+Description: Blocked time for Closures.
+
+##### ReceiverProcessTotalTime
+
+Description: Total time taken for receiver-side processing.
+
+##### WaitLockTime
+
+Description: Lock waiting time.
+
+### Aggregate Operator
+
+##### GroupingKeys
+
+Description: GROUP BY columns.
+
+##### AggregateFunctions
+
+Description: Time taken for aggregate function calculations.
+
+##### AggComputeTime
+
+Description: Time for AggregateFunctions + Group By.
+
+##### ChunkBufferPeakMem
+
+Description: Peak memory usage of the Chunk Buffer.
+
+##### ChunkBufferPeakSize
+
+Description: Peak size of the Chunk Buffer.
+
+##### ExprComputeTime
+
+Description: Time for expression computation.
+
+##### ExprReleaseTime
+
+Description: Time for expression release.
+
+##### GetResultsTime
+
+Description: Time to extract aggregate results.
+
+##### HashTableSize
+
+Description: Size of the Hash Table.
+
+##### HashTableMemoryUsage
+
+Description: Memory size of the Hash Table.
+
+##### InputRowCount
+
+Description: Number of input rows.
+
+##### PassThroughRowCount
+
+Description: In streaming mode, the amount of data processed due to low aggregation leading to degradation to streaming mode.
```

Comment:
the amount of data processed in streaming mode after low aggregation leads to degradation to streaming mode.